src/components/garage/__tests__/GarageSelector.test.tsx

```
@@ -1,124 +1,113 @@
-    (useNavigate as ReturnType<typeof vi.fn>).mockReturnValue(mockNavigate);
-    
-    // Setup Supabase mocks
-    (supabase.auth.getUser as ReturnType<typeof vi.fn>).mockResolvedValue(mockUser);
   });
 
-  it("should render loading state initially", () => {
-    renderWithProviders(<GarageSelector />);
-    expect(screen.getByRole("status")).toBeInTheDocument();
-  });
+  it('renders loading state', async () => {
+    const { useGarages } = require('../hooks/useGarages');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)